server/tests/controllers/test_filter_controller.py

```diff
@@ -808,10 +862,18 @@ async def test_filter_commits_bypassing_prs_no_with(client, headers):
                                                            tzinfo=dateutil.tz.tzutc())
 
 
+@pytest.mark.filter_commits
+@pytest.mark.parametrize("cached", [False, True], ids=["no cache", "with cache"])
 @pytest.mark.parametrize("account, date_to, code",
                          [(3, "2020-02-22", 403), (10, "2020-02-22", 403), (1, "2020-01-12", 200),
                           (1, "2010-01-11", 400), (1, "2020-02-32", 400)])
-async def test_filter_commits_bypassing_prs_nasty_input(client, headers, account, date_to, code):
+async def test_filter_commits_bypassing_prs_nasty_input(client, cached, headers, app, client_cache,
+                                                        account, date_to, code):
+    if cached:
```

Comment:
We need a JIRA task to write a decorator to avoid this repetition.

Reply:
DEV-497

Reply:
totally agree 💯